### ISSUE TITLE
Mover chat para o Slack

### ISSUE THREAD

**Issue by plribeiro3000** [RubyCastsBrasil/RubyCastsBrasil#203]
Modificar links para apontar para o canal do slack ao invés do gitter e mandar mensagem no canal do gitter avisando a todos da mudança.

**Comment by plribeiro3000** [comment]
O Slack não suporta organizações com cadastro aberto ou emails que terminam com `@gmail` portanto é inviável migrarmos para o `slack` nesse momento. Mas vou deixar essa issue em aberto porque vou entrar em contato com eles para descobrir se tem algum esquema para open source.

**Comment by evansantos** [comment]
eles aceitam sim, Paulo... tenho varias contas de gmail no slack, dá uma
perguntada pro pessoal do rubytalk como eles fizeram

-- 
Atenciosamente,
Evandro Santos


2015-04-12 22:31 GMT-03:00 Paulo Henrique Lopes Ribeiro <
notifications@github.com>:

**Comment by rrrcompagnoni** [comment]
também uso o gmail no slack, inclusive para criar orgs

**Comment by plribeiro3000** [comment]
Opa. Então. Eu uso o meu gmail lá também. Só deu zuera na hora de abrir para outras pessoas entrarem. Vou dar uma perguntada pra galera como eles fizeram. Valeu pela feedback galera. =)

**Comment by plribeiro3000** [comment]
Bom, acabei de achar isso. Acho que uma solução seria criar uma rota no site do rubycastsbrasil apra lidar com isso. O que acham?

**Comment by PedroVinicius** [comment]
Acho uma boa ideia a questão da rota @plribeiro3000

**Comment by plribeiro3000** [comment]
Vou ver se faco um rascunho disso agora a tarde pra ver. =)

**Comment by pragmaticivan** [comment]
Da pra issar isso aqui: https://github.com/seabre/inviteyoself

**Comment by plribeiro3000** [comment]
Hmmm. Maneiro. Acabei de receber um convite do http://prud.io também.
Ja que dou uma olhada nisso mais a fundo. =)

Se alguém quiser brincar com isso só me avisar. :p

**Comment by wenderjean** [comment]
Que conclusão tu chegou @plribeiro3000? Vai migrar?

**Comment by wenderjean** [comment]
Prud ae pareceu interessante.

**Comment by plribeiro3000** [comment]
Então. Preciso dar mais uma olhada. A primeira vista não saquei como faz com o prud, só consegui criar um esquema de chamado (tipo chat online).

**Comment by mfbmina** [comment]
@plribeiro3000 não rola especificar as vantagens e desvantagens de cada opção pra ficar mais fácil de opinar na escolha?

**Comment by plribeiro3000** [comment]
@mfbmina Na verdade não tem opção não. O prud não tem essa opção. Pelo menos não consegui achar.

E esse outro projeto que o @pragmaticivan passou eu não olhei mas parece bem sussa.

É isso que tenho a reportar. =s

**Comment by mfbmina** [comment]
Achei que criar uma rota, usar o prudio ou o que @pragmaticivan passou eram opções para utilizar o slack. Não é isso?

**Comment by pragmaticivan** [comment]
Como ficou isso aqui?

**Comment by plribeiro3000** [comment]
Acabei não mechendo mais. =s

**Comment by pragmaticivan** [comment]
Por mim usaria o https://github.com/seabre/inviteyoself. 

Alguém tem interesse em fazer essa task?

**Comment by plribeiro3000** [comment]
Caria, pode mim pode ser também. =)

Eu to meio atolado aki agora. =s

**Comment by wenderjean** [comment]
Eu faço, pode ser?

**Comment by pragmaticivan** [comment]
Ta pra você a task @wenderjean

**Comment by wenderjean** [comment]
Galera a aplicação está hoje no Heroku? Teremos que fazer um deploy do inviteyoself e uma rota apontando pra ele.

**Comment by plribeiro3000** [comment]
yep. Eu vou fazer isso agora a tarde sem erro. =)

**Comment by wenderjean** [comment]
Blz, criei um aqui pra testar, foi jogo rápido. :) Se precisar que faça algo vou estar on. O Team já existe no slack?

**Comment by plribeiro3000** [comment]
Opa, ta rodando bunitao jah?

A gente ja tem slack sim. =)

Quer fazer isso?

**Comment by wenderjean** [comment]
Então, eu fiz aqui rapidão pra conhecer melhor ele mas como o heroku não libera o schedule na versão free acabou não ficando completo.

https://morning-headland-2920.herokuapp.com/

Se tu tiver enrolado ae eu faço, só vou precisar das infos pra configurar. :)

**Comment by plribeiro3000** [comment]
Mas ele vai funcionar na versão free?

**Comment by plribeiro3000** [comment]
o @wenderjean ja resolveu isso. Agora so precisamos que o @pragmaticivan crie um subdominio e configure a app pra gnete poder linkar pra ela. =)

**Comment by plribeiro3000** [comment]
Parece que os sistema esta com alguns bugs. =s

**Comment by wenderjean** [comment]
Está com bugs mesmo, não está enviando os convites, tentei pela demo deles e tbm não recebi. Tem uma issue aberta lá com o mesmo problema, simplesmente aparece  "Failed!" sem muitas informações. Estou subindo uma VM aqui com Clojure pra fazer alguns testes locais, se não conseguir nada estou pensando em abrir outra issue lá.

**Comment by plribeiro3000** [comment]
Aew @wenderjean. Tem como tu colocar aqui o que foi que deu pau?

@pragmaticivan Entra la no heroku e seta um sudominio pra gente depois que tu configurar o DNS. =)

**Comment by wenderjean** [comment]
Então o slack alterou a forma que o inviteyoself utilizava para enviar os convites. Um colaborador enviou a correção lá mas não fizeram o merge então o  repo atual não está funcionando mesmo. O @plribeiro3000 fez o fork pra organização vou fazer a mesma correção e subir.

**Comment by wenderjean** [comment]
Pessoal, como tá isso aqui?

**Comment by plribeiro3000** [comment]
@pragmaticivan ?

Tem como tu adicionar o subdominio la pra gente no heroku depois de criar o record?

**Comment by pragmaticivan** [comment]
Qual a URL certa do Heroku?

**Comment by wenderjean** [comment]
invite-slack.herokuapp.com

**Comment by ribeiroevandro** [comment]
Os convites para o slack ainda funcionam? Acabei de testar aqui e não recebi email.

**Comment by plribeiro3000** [comment]
@ribeiroevandro Parece que realmente parou de funcionar. Precisamos achar outra alternativa para o mesmo. =s

**Comment by ribeiroevandro** [comment]
@plribeiro3000 qual o problema com o Gitter?

**Comment by plribeiro3000** [comment]
@ribeiroevandro Nenhum problema com o gitter especificamente. O maior ponto é que hoje em dia muita gente usa o slack daí se a gente usar o slack aqui também fica mais fácil pq ele possibilita conectar em vários canais ao mesmo tempo e a gente não precisa ter o cliente do gitter só para o rubycasts.
Mas se não tiver uma forma de resolver esse esquema do invite a melhor solução vai ser voltar para o gitter mesmo. =s

**Comment by ribeiroevandro** [comment]
@plribeiro3000 A comunidade Laravel tem um app feito com ele para isso, se quiser posso subir uma app no heroku para isso, o que acham

**Comment by plribeiro3000** [comment]
feito com o que, slack?

**Comment by ribeiroevandro** [comment]
Feito com Lumem micro-framework baseado no Laravel para invite do slack.

**Comment by plribeiro3000** [comment]
Bah. Acho que rola sim. =)

**Comment by ribeiroevandro** [comment]
A app já está online [invite](http://castsbrasil-invite.herokuapp.com) porém existe um erro na comunicação e preciso de acesso de administrador ao team no slack para verificar e fazer os devidos testes, após finalizar meu nível de acesso pode ser alterado.

**Comment by plribeiro3000** [comment]
@ribeiroevandro Acho que agora tu deve ter as informações que precisa. Me avise se precisar de mais alguma coisa.

Obrigado pela força!

**Comment by ribeiroevandro** [comment]
@plribeiro3000 acabei de ver o que me mandou no Gitter e já coloquei a app online, como não tinha o cadastro ainda, me usei como teste e o convite foi simultâneo... A app pode ser acessa por aqui [Casts Brasil Invite](http://castsbrasil-invite.herokuapp.com), o próximo passo seria atualizar o README do projeto, posso fazer isso ou devo aguardar?

**Comment by plribeiro3000** [comment]
Manda ver cara. O que tu conseguir mandar PR ou algo do tipo, pode fazer! 

Valeu!

**Comment by ribeiroevandro** [comment]
Estava pensando, provavelmente o canal do slack tenha seu nome e endereços alterados, até definir isso, não compensa enviar uma PR agora para mais pra frente fazer uma nova corrigindo a correção.

**Comment by plribeiro3000** [comment]
verdade. Esse fds vou terminar de migrar as coisas e adicionar no README que a gente migrou.
Ainda tem a comunidade no face e talz.

**Comment by juuh42dias** [comment]
Como está o andamento dssa issue? O grupo foi criado? @plribeiro3000
